src/okteto.ts

```diff
@@ -273,17 +276,28 @@ export function showTerminal(){
   });
 }
 
-export function getOktetoId(): string | undefined {
+export function getOktetoId(): {id: string, machineId: string} {
   const tokenFile =  path.join(home, oktetoFolder, ".token.json");
+  let oktetoId: string ;
```

Comment:
nit: remove extra space
```suggestion
  let oktetoId: string;
```